openedx/adg/lms/applications/models.py

```diff
@@ -341,30 +346,76 @@ def get_user_enrolled_course(self, user):
         Returns:
             CourseOverview: Enrolled multilingual course or None
         """
-        enrolled_multilingual_course = self.multilingual_courses(
-            manager='open_multilingual_courses'
-        ).filter(
+        if user.is_anonymous:
+            return None
```

Comment:
This method is responsible to return enrolled course from a group for a user, So for anonymous user it will return None, The method ```get_courses``` will return the list of courses for a user.

Reply:
Yes, I was particularly asking if we can omit this check and the the filter below could server the purpose, but we can't, I've validated on my machine.